src/BankAccountApp/CreateAccountsFromFile.java

```diff
@@ -0,0 +1,56 @@
+package BankAccountApp;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+// read a CSV File then create new accounts based on that data
+
+class CreateAccountsFromFile {
+
+
+    ArrayList<Account> createAccountsBatch() {
```

Comment:
There's no need to create an instance of `CreateAccountsFromFile` to execute this method. You could convert this instance method to a static method such as `public static ArrayList<Account> createAccountsBatch() { ... }` where `...` is the content of this method. Users of this method would change their invocation to something like: `CreateAccountsFromFile.createAccountsBatch()`

In general, classes that start with a verb tend to be better as functions rather than objects but I understand you were separating code here

Reply:
Oh....thank you! It felt unnecessary, but once I separated it into its own class, I couldn't think of how to access the method without creating an instance. Much better, thanks!